record-and-playback/core/scripts/post_events/post_events_analytics_callback.rb

```
@@ -173,12 +173,8 @@ def format_analytics_data!(data)
     events_data = BBBEvents.parse(events_xml_path)
 
     # Write JSON data to file.
-    File.open(data_json_path, 'w') do |f|
-      f.write(events_data.to_json)
-    end
-
-    json_file = File.open(data_json_path)
-    data = JSON.load(json_file)
+    File.write(data_json_path, events_data.to_json)
+    data = JSON.load(File.read(data_json_path))
```

RuboCop (1.86.2)

[convention] Prefer `JSON.parse` over `JSON.load`.

(Security/JSONLoad)